setup.py

```diff
@@ -33,10 +33,10 @@
 ]
 
 if PY3:
-    install_requires.append("Twisted >= 15.5.0")
+    install_requires.append("Twisted >= 16.0.0")
```

Comment:
Okay, I addressed this in #138.  Perhaps it should be a separate PR.  Let me know.